src/test/scala/org/broadinstitute/dsde/firecloud/model/SamResourceSpec.scala

```diff
@@ -7,7 +7,6 @@ import spray.json._
 import org.broadinstitute.dsde.firecloud.model.ModelJsonProtocol._
 import spray.json.DefaultJsonProtocol._
 
-import scala.util.parsing.json.JSONFormat
```

Comment:
this import was unused, and caused compilation problems. Instead of figuring out why, I just removed it.